lib/src/connected_device_operation.dart

```diff
@@ -67,25 +66,25 @@ class ConnectedDeviceOperationImpl implements ConnectedDeviceOperation {
   @override
   Future<void> writeCharacteristicWithoutResponse(
     QualifiedCharacteristic characteristic, {
-    @required List<int> value,
+    required List<int> value,
   }) async =>
       _controller
           .writeCharacteristicWithoutResponse(characteristic, value)
           .then((info) => info.result.dematerialize());
 
   @override
-  Stream<List<int>> subscribeToCharacteristic(
+  Stream<List<int>?> subscribeToCharacteristic(
```

Comment:
I repeat and insist: this optionality is suspicious and should be eliminated (I checked the implementation and don't see a straightforward reason why it's needed) in this and similar cases, unless you can explain _why it makes sense_.